volontulo/tests/views/test_offers.py

```diff
@@ -137,3 +137,96 @@ def test_offers_list_for_admin(self):
         self.assertIn('offers', response.context)
         # pylint: disable=no-member
         self.assertEqual(len(response.context['offers']), 2)
+
+
+class TestOffersCreate(TestCase):
+    u"""Class responsible for testing offer's create page."""
+
+    @classmethod
+    def setUpTestData(cls):
+        u"""Set up data for all tests."""
+        organization = Organization.objects.create(
+            name=u'',
+            address=u'',
+            description=u'',
+        )
+        organization.save()
+        organization_user = User.objects.create_user(
+            u'organization@example.com',
+            u'organization@example.com',
+            u'123org'
+        )
+        organization_user.save()
+        cls.organization = UserProfile(
+            user=organization_user,
+        )
+        cls.organization.save()
+        # pylint: disable=no-member
+        cls.organization.organizations.add(organization)
```

Comment:
This take me awhile to understand this piece of code. Why you didn't use cls.userprofile or even better cls.organization_profile?